lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java

```diff
@@ -600,4 +603,38 @@ void should_NotThrowServiceLevelViolation_IfWeekendOrHolidaysBetweenDates() thro
     assertThat(task.getDue()).isEqualTo(getInstant("2020-04-14T07:00:00")); // Tuesday
     assertThat(task.getPlanned()).isEqualTo(getInstant("2020-04-09T07:00:00")); // Thursday
   }
+
+  @WithAccessId(user = "admin")
```

Comment:
Do we need admin for this? As far as i can see there is no role check in that methd (setPlannedPropertyOfTasks)

Reply:
Actually I didn't change that. However I can image that this is required to make sure that all tasks are updated and not just those, the user has access to. 'taskadmin' might be an alternative user to do that. If this is important to you?